packages/validator/test/unit/services/indicesService.test.ts

```diff
@@ -0,0 +1,60 @@
+import {toBufferBE} from "bigint-buffer";
+import {expect} from "chai";
+import sinon from "sinon";
+import bls from "@chainsafe/bls";
+import {toHexString} from "@chainsafe/ssz";
+import {ValidatorStore} from "../../../src/services/validatorStore";
+import {getApiClientStub} from "../../utils/apiStub";
+import {testLogger} from "../../utils/logger";
+import {IndicesService} from "../../../src/services/indices";
+
+describe("IndicesService", function () {
+  const sandbox = sinon.createSandbox();
+  const logger = testLogger();
+  const api = getApiClientStub(sandbox);
+  const validatorStore = sinon.createStubInstance(ValidatorStore) as ValidatorStore &
+    sinon.SinonStubbedInstance<ValidatorStore>;
+
+  let pubkeys: Uint8Array[]; // Initialize pubkeys in before() so bls is already initialized
+
+  before(() => {
+    const secretKeys = [
+      bls.SecretKey.fromBytes(toBufferBE(BigInt(98), 32)),
+      bls.SecretKey.fromBytes(toBufferBE(BigInt(99), 32)),
+    ];
+    pubkeys = secretKeys.map((sk) => sk.toPublicKey().toBytes());
+  });
+
+  it("Should remove pubkey", async function () {
+    const indicesService = new IndicesService(logger, api, validatorStore);
+    const firstValidatorIndex = 0;
+    const secondValidatorIndex = 1;
+
+    const pubkey1 = toHexString(pubkeys[firstValidatorIndex]);
+    const pubkey2 = toHexString(pubkeys[secondValidatorIndex]);
+
+    indicesService.index2pubkey.set(firstValidatorIndex, pubkey1);
+    indicesService.index2pubkey.set(secondValidatorIndex, pubkey2);
+
+    indicesService.pubkey2index.set(pubkey1, firstValidatorIndex);
+    indicesService.pubkey2index.set(pubkey2, secondValidatorIndex);
+
+    // remove pubkey2
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-call
```

Comment:
Why do you need to suppress `@typescript-eslint/no-unsafe-call`? Isn't this typed? I've seen this lint rule suppression used in many places in this PR, please remove from everywhere it must not be necessary-

Reply:
<img width="492" alt="Screenshot 2022-02-22 at 11 24 01" src="https://user-images.githubusercontent.com/272535/155112913-1a3ed2ee-30ac-4e35-9c56-616f6f16e480.png">

I do get the red lines even in places where the variable is typed, and should not show up. Maybe an issue with IDE integration or somewhere else. Anyways I'll remove the suppress comments